site/doc/config/reference.md

```diff
@@ -254,6 +254,20 @@ In this case the run step name will be the same of the command trimmed at the ma
 | environment | Map: EnvVarName(String) => EnvVarValue([Value](#value)) | Environment variables to set                                                       |
 | working_dir | String                                                  | Working dir where the entrypoint will be executed (also used for steps in a task)  |
 | user        | String                                                  | The user id or username to use when executing the entrypoint or the task run steps |
+| volumes     | List: [Volume](#Volume)                                 | A list of volumes                                                                  |
+
+###### Volume
+
```

Comment:
We should add more details and examples on how volume works and that now only tmpfs is supported that it must be provided